MC Server automatic version picker YAD prompt

theofficialgman opened the pull request:
Fabric, Forge, Vanilla, Paper, Bukkit, Spigot, Purpur, and Tuinity are ALL supported. The old system of opening the web browser and having the user choose the appropriate java version has been removed, everything is entirely automated.

this also automatically pics the version of java necessary for everything and autoselects it instead of asking the user

to close https://github.com/Botspot/pi-apps/pull/1406

Crilum commented on apps/Minecraft Java Server/install:
```diff
@@ -83,16 +88,44 @@ done
+  description="In order to run Minecraft, the game requires a specific version of Java.\
+  \nPlease choose below which version of java the server of your choice should run with.\
+  \nJava versions can be 8, 16, or 17.\
+  \n\nAll version of Minecraft before and including Minecraft 1.16.5 should use Java 8.\
```
```suggestion
  \n\nAll versions of Minecraft before and including Minecraft 1.16.5 should use Java 8.\
```
Just a typo ;)

theofficialgman commented:
fabric has a decent meta api, going to modify this script to include it so users don't need to use the website. don't merge yet

theofficialgman commented:
🥳 ALL server types now support downloading directly through the install script
there is no more web browser opening or manual user file placing left in this script

@KolpixTV consider your suggestion fully implemented now

rbcoder117 commented:
@Botspot

theofficialgman commented:
just rebased and cleaned up commit messages a bit. no changes.
I consider this a bugfix PR with some important feature updates. Since the MC Server script seems to be getting some use, going to go ahead and merge myself
